docs/istio-on-karmada.md

```diff
@@ -0,0 +1,457 @@
+```
+
+### Prepare member2 cluster secret
+
+1. Export `KUBECONFIG` and switch to `karmada member2`:
+```bash
+export KUBECONFIG="$HOME/.kube/members.config"
+kubectl config use-context member2
+```
+
+2. Create istio remote secret for member1:
+```bash
+istioctl x create-remote-secret --name=member2 > istio-remote-secret-member2.yaml
+```
+
+### Apply istio remote secret
+
+Export `KUBECONFIG` and switch to `karmada apiserver`:
+
+```
+# export KUBECONFIG=$HOME/.kube/karmada.config
+
+# kubectl config use-context karmada-apiserver 
+```
+
+Apply istio remote secret:
+```bash 
+kubectl apply -f istio-remote-secret-member1.yaml
+
+kubectl apply -f istio-remote-secret-member2.yaml
+```
+
+
+### Install istio remote
+
+1. Install istio remote member1
+
+Export `KUBECONFIG` and switch to `karmada member1`:
+```bash
+export KUBECONFIG="$HOME/.kube/members.config"
+kubectl config use-context member1
+```
+
+```bash
+cat <<EOF | istioctl install -y -f -
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: member1
+      network: network1
+      remotePilotAddress: ${DISCOVERY_ADDRESS}
+EOF
+```
+
+2. Install istio remote member2
+
+Export `KUBECONFIG` and switch to `karmada member2`:
+```bash
+export KUBECONFIG="$HOME/.kube/members.config"
+kubectl config use-context member2
+```
+
+```bash
+cat <<EOF | istioctl install -y -f -
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: member2
+      network: network1
+      remotePilotAddress: ${DISCOVERY_ADDRESS}
+EOF
+```
+
+## Deploy bookinfo application
+
+Export `KUBECONFIG` and switch to `karmada apiserver`:
+
+```
+# export KUBECONFIG=$HOME/.kube/karmada.config
+
+# kubectl config use-context karmada-apiserver 
+```
+
+Create an `istio-demo` namespace:
+
+```bash
+kubectl create namespace istio-demo
+```
+
+Label the namespace that will host the application with `istio-injection=enabled`:
+```bash
+kubectl label namespace istio-demo istio-injection=enabled
+```
+
+Deploy your application using the `kubectl` command:
+```bash
+kubectl apply -nistio-demo -f https://raw.githubusercontent.com/istio/istio/release-1.12/samples/bookinfo/platform/kube/bookinfo.yaml
+```
+
+Run the following command to create default destination rules for the Bookinfo services:
+```bash
+kubectl apply -nistio-demo -f https://raw.githubusercontent.com/istio/istio/release-1.12/samples/bookinfo/networking/destination-rule-all.yaml
+```
+
+Run the following command to create virtual service for the Bookinfo services:
+```bash
+kubectl apply -nistio-demo -f https://raw.githubusercontent.com/istio/istio/release-1.12/samples/bookinfo/networking/virtual-service-all-v1.yaml
+```
+
+Run the following command to create propagation policy for the Bookinfo services:
```

Comment:
Suggest drawing a diagram to show the deployment topology

Reply:
Updated the arch image